src/wrappers/express.js

```diff
@@ -30,7 +31,11 @@ function expressMiddleware(req, res, next) {
     tracer.restart();
     let expressEvent;
     const startTime = Date.now();
+    const epsagonIdentifier = uuid4();
     try {
+        // Add epsagon id to
+        req.epsagonId = epsagonIdentifier;
+        tracer.getTrace = () => traceContext.get(epsagonIdentifier);
```

Comment:
this will be problematic in a multi request environment - tracer is a singleton!

Reply:
everything in this case must be managed on the request instead of in the singleton.

Reply:
This line should be instead of line 111